feature/RA-improvement

ShilpiMaurya opened the pull request:
# Changes done
- Improved the design of the website.

vaibhav-kumar-tokyo commented on src/resume.json:
```diff
@@ -1,8 +1,8 @@
-    "age": "24 Years Old",
-    "label": "Aspiring Front-end Developer",
+    "age": "25 years old",
+    "label": "Aspiring front-end developer",
```
can remove `Aspiring`, you are a front-end dev now :)

vaibhav-kumar-tokyo commented on src/resume.json:
```diff
@@ -21,20 +21,39 @@
+        "institution": "Udemy (online course)",
+        "area": "JavaScript algorithms and data structures masterclass",
+        "startDate": "2020 / January",
+        "endDate": "on going",
```
`ongoing` is a single word, right?

vaibhav-kumar-tokyo commented on src/resume.json:
```diff
@@ -21,20 +21,39 @@
+        "startDate": "2013 / August",
         "endDate": "2017 / June",
-        "courses": "Electrical machines - Switch gear and protection - Human Values and Professional Ethics - Cyber Security"
+        "courses": "electrical machines, switch gear and protection, human Values and professional ethics, cyber security"
```
Values -> values

vaibhav-kumar-tokyo replied:
can add any programming or electronic courses here, as they contribute to your dev skills or technical thinking or problem solving

vaibhav-kumar-tokyo commented:
![image](https://user-images.githubusercontent.com/6745332/84399198-a89b0100-ac3b-11ea-967a-49dc4b2242f4.png)
padding missing

vaibhav-kumar-tokyo commented:
Can not really ready some skills:
![image](https://user-images.githubusercontent.com/6745332/84399467-dd0ebd00-ac3b-11ea-9336-bb09ee4a1e71.png)

even when we have space in mobile view
![image](https://user-images.githubusercontent.com/6745332/84399521-ec8e0600-ac3b-11ea-8edc-c6e98a53d960.png)

vaibhav-kumar-tokyo commented:
> Can not really ready some skills:
> ![image](https://user-images.githubusercontent.com/6745332/84399467-dd0ebd00-ac3b-11ea-9336-bb09ee4a1e71.png)
> 
> even when we have space in mobile view
> ![image](https://user-images.githubusercontent.com/6745332/84399521-ec8e0600-ac3b-11ea-8edc-c6e98a53d960.png)

Do you wanna try putting skills in pills and remove the skill-level (Intermediate, Beginner etc.)?

ShilpiMaurya commented:
# Skill section in pill view
<img width="1324" alt="Screen Shot 2020-06-11 at 21 10 36" src="https://user-images.githubusercontent.com/47273243/84407162-13424180-ac28-11ea-8b09-5385b82be0b6.png">

ShilpiMaurya commented:
# Skill section in pill view
<img width="1374" alt="Screen Shot 2020-06-11 at 21 31 52" src="https://user-images.githubusercontent.com/47273243/84409496-21de2800-ac2b-11ea-8019-4533fb104199.png">

vaibhav-kumar-tokyo commented:
Pls remove Eslint and prettier and  CSS animation

vaibhav-kumar-tokyo commented:
> Pls remove Eslint and prettier and CSS animation

instead you could write `next.js` and `gatsby`

vaibhav-kumar-tokyo commented:
Also, profile image can be changed to one you are using on medium, it looks more professional.

vaibhav-kumar-tokyo commented on src/Navigation/Skill/index.css:
```diff
@@ -53,12 +53,18 @@
   align-items: center;
   align-content: center;
-  font-size: 18px;
+  /* font-size: 18px; */
```
remove the comment

vaibhav-kumar-tokyo commented:
Font size of skill-pills seem too big.

ShilpiMaurya commented:
# Modified skill section

## With different screen sizes

<img width="728" alt="Screen Shot 2020-06-11 at 23 29 05" src="https://user-images.githubusercontent.com/47273243/84423116-a638a700-ac3b-11ea-8141-b8a1bca15af8.png">

<img width="497" alt="Screen Shot 2020-06-11 at 23 29 27" src="https://user-images.githubusercontent.com/47273243/84423154-ae90e200-ac3b-11ea-8f56-f48b44c0adcd.png">

vaibhav-kumar-tokyo commented:
Two more changes:
1. Too much padding on pills.
2. skills section has uneven spacing in start and end, as shown here
![image](https://user-images.githubusercontent.com/6745332/84481823-a8e5db80-acd1-11ea-943c-c7f1d21fd789.png)

vaibhav-kumar-tokyo commented:
> ![image](https://user-images.githubusercontent.com/6745332/84399198-a89b0100-ac3b-11ea-967a-49dc4b2242f4.png)
> padding missing

this is not fixed yet

vaibhav-kumar-tokyo commented:
[Suggestion] Regarding pills width, let's try giving them a `min-width` instead of `width` and see if it looks better?

ShilpiMaurya commented:
# Improvement in Pills design

## With different screen sizes

<img width="310" alt="Screen Shot 2020-06-12 at 17 41 44" src="https://user-images.githubusercontent.com/47273243/84501552-50fca400-acd4-11ea-819f-576dd23445bd.png">

<img width="994" alt="Screen Shot 2020-06-12 at 17 42 42" src="https://user-images.githubusercontent.com/47273243/84501572-58bc4880-acd4-11ea-9b47-b2624b485b91.png">

ShilpiMaurya commented:
> > ![image](https://user-images.githubusercontent.com/6745332/84399198-a89b0100-ac3b-11ea-967a-49dc4b2242f4.png)
> > padding missing
> 
> this is not fixed yet

>>I didn't understand this comment whether  padding around whole nav bar or individual blocks.

vaibhav-kumar-tokyo commented:
> > > ![image](https://user-images.githubusercontent.com/6745332/84399198-a89b0100-ac3b-11ea-967a-49dc4b2242f4.png)
> > > padding missing
> > 
> > 
> > this is not fixed yet
> 
> > > I didn't understand this comment whether  padding around whole nav bar or individual blocks.

This whole section, in mobile view the text hugs the viewport's boundary.

vaibhav-kumar-tokyo commented on src/Navigation/Skill/index.css:
```diff
@@ -53,12 +53,17 @@
+  text-overflow: ellipsis;
+  white-space: nowrap;
+  overflow: hidden;
+  padding: 5px;
```
```suggestion
  padding: 6px 12px;
```

vaibhav-kumar-tokyo commented on src/Navigation/Skill/index.css:
```diff
@@ -78,6 +83,10 @@
   justify-content: center;
   align-content: center;
   align-items: center;
+  transition: all 0.5s ease-in-out;
```
on line 81: change padding to 12px

vaibhav-kumar-tokyo commented:
Also, pls check your site on safari

ShilpiMaurya commented:
# Latest look of Skill section and nav bar
## Changes done
- Adjusted padding of pills
- Gave pills some shadow

<img width="686" alt="Screen Shot 2020-06-13 at 18 36 18" src="https://user-images.githubusercontent.com/47273243/84569794-a1dccd00-ada6-11ea-8f2c-13bb6497994b.png">

## Changes done
- Adjusted size of nav-bars

<img width="673" alt="Screen Shot 2020-06-13 at 18 36 02" src="https://user-images.githubusercontent.com/47273243/84569797-aef9bc00-ada6-11ea-8e04-fa0206b4f65d.png">

<img width="732" alt="Screen Shot 2020-06-13 at 18 54 54" src="https://user-images.githubusercontent.com/47273243/84569902-6f7f9f80-ada7-11ea-913e-83d243abb33c.png">

ShilpiMaurya commented:
> Also, pls check your site on safari
>Have to fix this problem

<img width="754" alt="Screen Shot 2020-06-13 at 18 58 15" src="https://user-images.githubusercontent.com/47273243/84569989-faf93080-ada7-11ea-9663-4d92fa309820.png">

vaibhav-kumar-tokyo commented:
please commit your changes so that I can check

vaibhav-kumar-tokyo commented:
> > > > ![image](https://user-images.githubusercontent.com/6745332/84399198-a89b0100-ac3b-11ea-967a-49dc4b2242f4.png)
> > > > padding missing
> > > 
> > > 
> > > this is not fixed yet
> > 
> > 
> > > > I didn't understand this comment whether  padding around whole nav bar or individual blocks.
> 
> This whole section, in mobile view the text hugs the viewport's boundary.

also, did you fix this issue?

ShilpiMaurya commented:
> > > > > ![image](https://user-images.githubusercontent.com/6745332/84399198-a89b0100-ac3b-11ea-967a-49dc4b2242f4.png)
> > > > > padding missing
> > > > 
> > > > 
> > > > this is not fixed yet
> > > 
> > > 
> > > > > I didn't understand this comment whether  padding around whole nav bar or individual blocks.
> > 
> > 
> > This whole section, in mobile view the text hugs the viewport's boundary.
> 
> also, did you fix this issue?

<img width="332" alt="Screen Shot 2020-06-13 at 19 25 48" src="https://user-images.githubusercontent.com/47273243/84570534-c1c2bf80-adab-11ea-8c24-864ee8245521.png">

vaibhav-kumar-tokyo commented on src/Navigation/NavBar/index.css:
```diff
@@ -8,25 +8,35 @@
   font-weight: 400;
   outline: none;
+  -moz-box-shawdow: var(--shadow-style);
+  -webkit-box-shadow: var(--shadow-style);
```
can try using drop-shadow filter too
https://developer.mozilla.org/en-US/docs/Web/CSS/filter-function/drop-shadow
